src/main/resources/export/java/build.gradle

```diff
@@ -0,0 +1,58 @@
+plugins {
+    id "java"
+    id "eclipse"
+    id "idea"
+    id "jaci.openrio.gradle.GradleRIO" version "2018.01.22"
+}
+
+#@autogenerated_code("team-info", "")
+def TEAM = ${team-number}
+def ROBOT_CLASS = "${package}.${robot-name}"
+#end
+
+if (project.hasProperty('teamNumber')) {
+    TEAM = teamNumber.toInteger()
+}
+
+def useDebug = false
+
+if(project.hasProperty('debugMode')) {
+    useDebug = true;
+}
+
+// Define my targets (RoboRIO) and artifacts (deployable files)
+// This is added by GradleRIO's backing project EmbeddedTools.
+deploy {
+    targets {
+        target("roborio", jaci.openrio.gradle.frc.RoboRIO) {
+            team = TEAM
+        }
+    }
+    artifacts {
+        artifact('frcJava', jaci.openrio.gradle.frc.FRCJavaArtifact) {
+            targets << "roborio"
+            debug = useDebug
+        }
+    }
+}
+
+// Defining my dependencies. In this case, WPILib (+ friends), CTRE Phoenix (Talon SRX)
+// and NavX.
+dependencies {
+    compile wpilib()
+    compile ctre()
+    compile navx()
+    compile openrio.powerup.matchData()
+}
+
+// Setting up my Jar File. In this case, adding all libraries into the main jar ('fat jar')
+// in order to make them all available at runtime. Also adding the manifest so WPILib
+// knows where to look for our Robot Class.
+jar {
+    from configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
+    manifest jaci.openrio.gradle.GradleRIOPlugin.javaManifest(ROBOT_CLASS)
+}
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '4.4'
+}
```

Comment:
This is depreciated in the new versions of Gradle. You want to configure the existing task instead of creating a new one.